app/next-client-app/components/homepage/bentoProjects.tsx

```diff
@@ -0,0 +1,81 @@
+import { BentoCard, BentoGrid } from "@/components/magicui/bento-grid";
+import BoxReveal from "@/components/magicui/box-reveal";
+
+const features = [
+  {
+    name: "NOTTINGHAM BRC",
+    description:
+      "The Nottingham BRC has leveraged Carrot in its mission to advance health informatics by driving translational research and innovation.",
+    background: (
+      <img className="opacity-80 p-5 dark:bg-white" src="/logos/nbrc.png" />
+    ),
+    href: "https://nottinghambrc.nihr.ac.uk/research/informatics",
+    cta: "Learn more",
+    className: "lg:col-start-1 lg:col-end-4 lg:col-span-3",
+  },
+  {
+    name: "ALLEVIATE",
+    description:
+      "Alleviate has utilized Carrot to support the creation of its UK-wide pain data hub in collaboration with Health Data Research UK (HDR UK).",
+    background: (
+      <img className="opacity-80 p-5 w-[300px]" src="/logos/alleviate.jpeg" />
+    ),
+    href: "https://alleviate.ac.uk/",
+    cta: "Learn more",
+    className: "lg:col-end-7 lg:col-span-3",
+  },
+  {
+    name: "CO-CONNECT",
+    description:
+      "CO-CONNECT used Carrot to streamline COVID-19 data access and integration across the UK, aiding researchers in developing therapies and treatments.",
```

Comment:
```suggestion
      "CO-CONNECT developed Carrot to streamline COVID-19 data access and integration across the UK, aiding researchers in developing therapies and treatments.",
```
(if there is an order, this one should be first)